startup.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+
+# Install HomeBrew if not installed 
+if ! command -v brew >/dev/null 2>&1
+then   
+    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+else
+    echo "Home brew is installed"
+fi 
+
+# Install jq if not installed
+if ! command -v jq >/dev/null 2>&1
+then
+    brew install jq 
+else
+    echo "JQ is installed"
+fi 
+
+# Install Twilio CLI if not installed 
+if ! command -v twilio >/dev/null 2>&1
+then
+    brew tap twilio/brew && brew install twilio
+else
+    echo "Twilio cli is installed"
+fi 
+
+# Set Credentials for making Twilio API calls
+while getopts ":a:b:c:" OPTION 
+do
+    case $OPTION in
+        a) 
+            echo "account sid present"
+            TWILIO_ACCOUNT_SID="$OPTARG"
+            export TWILIO_ACCOUNT_SID=$TWILIO_ACCOUNT_SID
+            ;;
+        b)
+            echo "api key is present"
+            TWILIO_AUTH_TOKEN="$OPTARG"
+            export TWILIO_AUTH_TOKEN=$TWILIO_AUTH_TOKEN
+            ;;
+        c)
+            echo "sendgrid api key present"
+            SENDGRID_API_KEY="$OPTARG"
+            ;;
+        ?)
+            echo "You did not provide anything"
+            exit 1
+            ;;
+    esac
+done
+
+
+echo "Now creating needed services"
```

Comment:
NIT: be a little more specific "Now creating the Twilio services required for demo platform"